### PR TITLE
 HV-1593  HV-1594 Assorted Karaf features improvements

### PR DESCRIPTION
**Pull request by gsmet**
* https://hibernate.atlassian.net/browse/HV-1593
 * https://hibernate.atlassian.net/browse/HV-1594

### CODE REVIEW

File: osgi/karaf-features/src/main/features/features.xml [hunk 1]
```diff
@@ -17,10 +17,23 @@
...
+    </feature>
+    <feature name="hibernate-validator-jsoup" version="${project.version}">
+        <feature>hibernate-validator</feature>
         <bundle>mvn:org.jsoup/jsoup/${jsoup.version}</bundle>
+    </feature>
+    <feature name="hibernate-validator-joda-time" version="${project.version}">
+        <feature>hibernate-validator</feature>
         <bundle>mvn:joda-time/joda-time/${joda-time.version}</bundle>
+    </feature>
+    <feature name="hibernate-validator-javax-money" version="${project.version}" description="Be aware that the JavaMoney API license is currently non-free">
```
Comment (gunnarmorling) [inline osgi/karaf-features/src/main/features/features.xml]: I don't think we need this comment there. Not feeling like having a discussion of what's free and what's not in a feature description.

Comment (gsmet) [reply osgi/karaf-features/src/main/features/features.xml]: As you wish. I personally think it's important to raise awareness of that but I can remove it for sure.

Comment (gsmet) [reply osgi/karaf-features/src/main/features/features.xml]: @gunnarmorling force-pushed an update removing the description.

**Comment by gunnarmorling** [comment]
Thx, @gsmet!

**Pull request merged**